mocked_server/grpc_metrics/metrics_handler_server.js

```diff
@@ -0,0 +1,44 @@
+
+var PROTO_PATH = __dirname + '/metrichandler.proto';
+
+var grpc = require('@grpc/grpc-js');
+var protoLoader = require('@grpc/proto-loader');
+var packageDefinition = protoLoader.loadSync(
+    PROTO_PATH,
+    {keepCase: true,
+        longs: String,
+        enums: String,
+        defaults: true,
+        oneofs: true
+    });
+var data_handler_proto = grpc.loadPackageDefinition(packageDefinition).opentelemetry.proto.collector.metrics.v1;
+var get_data = ""
+
+/**
+ * Implements the Export RPC method.
+ */
+function Export(call, callback) {
+    get_data = "success";
+    callback(null, {message: "Export Data!"});
+}
+
+/**
+ * Implements the CheckData RPC method.
+ */
+function CheckData(call, callback) {
+    callback(null, {message: get_data});
+}
+
+/**
+ * Starts an RPC server that receives requests for the data handler service at the
+ * sample server port
+ */
+function main() {
```

Comment:
can this server also expose a http port 8080 as well so that we don't need to write code in validator to call grpc?

Reply:
to get validator to access the grpc from public internet, we will need to create a lb in front of it. if we can have the 8080 http port, then we don't need to change the code in ecs and eks